src/Avalonia.Themes.Fluent/Accents/BaseDark.xaml

```diff
@@ -2,16 +2,6 @@
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:sys="clr-namespace:System;assembly=netstandard">
   <Style.Resources>
-    <!-- Accent Colours -->
```

Comment:
BTW... I think we need to keep these... and they get overridden.

Using OS accent colours in UWP I believe can be turned on or off, falling back to some default ones..

Perhaps we can all user to define.... just SystemAccentColor... and the shades are generated.

Reply:
Well, will I undo the deleted comments?